### PR TITLE
Updated structure as discussed

### PR DESCRIPTION
**Pull request by concordium-cl**
## Purpose

Updated structure of page, mainly to include the new native distributions and group them all together under node distributions.

## Changes

- new maon node heading
- new Windows node
- new Mac node
- applied same format to mobile wallets with subheadings
- 
## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by concordium-cl** [comment]
I've restructured the page as discussed previously. Maybe we need some more user guidance for the node distributions in general. And all text and links etc are missing for the new two native distributions.

File: source/testnet/net/installation/downloads.rst [hunk 1]
```diff
@@ -11,20 +11,24 @@ Downloads
 
 This topics contains information about where you can download the Concordium tools and Concordium Wallets for |Net|. You can also find out about the hardware requirements for running a node.
```
Comment (LotteKh) [inline source/testnet/net/installation/downloads.rst:12]:
```suggestion
This topic contains information about where you can download the Concordium tools and Concordium Wallets for |Net|. You can also find out about the hardware requirements for running a node.
```

File: source/testnet/net/installation/downloads.rst [hunk 2]
```diff
@@ -83,17 +89,26 @@ Download cargo-concordium:
 -  `Download cargo-concordium for Windows <https://distribution.concordium.software/tools/windows/cargo-concordium_1.0.0.exe>`_
 
 
+Concordium Node Distributions
```
Comment (LotteKh) [inline source/testnet/net/installation/downloads.rst:92]:
```suggestion
Concordium node distributions
```

**Review by LotteKh — APPROVED** [review]

File: source/testnet/net/installation/downloads.rst [hunk 3]
```diff
@@ -83,17 +89,36 @@ Download cargo-concordium:
...
 
-Full suite for running a node using Docker
-===========================================
+Full suite for running a node on Linux using Docker
+---------------------------------------------------
 Download the full suite for running a node on Linux using Docker. The suite contains Concordium Node, Concordium Client and cargo-concordium.
 
 - `Download the suite for Linux <https://distribution.testnet.concordium.com/tools/linux/concordium-software-linux-1.0.1-testnet.tar.gz>`_
 
+Native Windows node
```
Comment (LotteKh) [inline source/testnet/net/installation/downloads.rst:108]:
```suggestion
Native Windows node
------------------------
```

File: source/testnet/net/installation/downloads.rst [hunk 4]
```diff
@@ -83,17 +89,36 @@ Download cargo-concordium:
...
+
+To run a node on Windows, you need a Windows Installer package.
+
+- `Download the Windows Installer package<https://distribution.concordium.software/windows/Signed/Node-1.1.1.msi>`
+-------------------
+
```
Comment (LotteKh) [inline source/testnet/net/installation/downloads.rst:115]:
```suggestion
- `Download the Windows Installer package<https://distribution.concordium.software/windows/Signed/Node-1.1.1.msi>`

```

File: source/testnet/net/installation/downloads.rst [hunk 5]
```diff
@@ -83,17 +89,36 @@ Download cargo-concordium:
...
+To run a node on Windows, you need a Windows Installer package.
+
+- `Download the Windows Installer package<https://distribution.concordium.software/windows/Signed/Node-1.1.1.msi>`
+
+Native Mac node
+----------------
+
+To run a node on macOS, you need a macOS installer package.
+
+- `Download the macOS installer package<https://distribution.concordium.software/macos/concordium-node-1.1.1.pkg>
```
Comment (LotteKh) [inline source/testnet/net/installation/downloads.rst:120]:
```suggestion
- `Download the macOS installer package<https://distribution.concordium.software/macos/concordium-node-1.1.1.pkg>`
```

**Pull request merged**